### PR TITLE
Remove WDN references

### PR DESCRIPTION
**Pull request by jsturek**

### CODE REVIEW

File: views/admin/events.erb [hunk 1]
```diff
@@ -16,17 +16,16 @@
         </div>
     </nav>
     <% end %>
-    <div class="<%= event_div_class %>">
-        <div id="pagetitle">
+    <div class="dcf-tabs-faked <%= event_div_class %>">
```
Comment (skoolbus39) [inline views/admin/events.erb:19]: Is there CSS or JS for this `dcf-tabs-faked` class? I couldn't find any.

Comment (jsturek) [reply views/admin/events.erb:19]: It's just an identifier to so tabs is not generated by javascript (if had class dcf-tabs) and to show that it is not intended to be.

Comment (jsturek) [reply views/admin/events.erb:19]: The markup used what javascript would have been run.  The way these tabs work is not supported by our DCF tabs javascript.

**Review by skoolbus39 — APPROVED** [review]

**Comment by jsturek** [comment]
@skoolbus39 I updated a lot more pages, please have another look.

File: views/home.erb [hunk 2]
```diff
@@ -2,18 +2,16 @@
...
 
-<div id="pagetitle">
-	<h3>Welcome to Innovation Studio Manager</h3>
-</div>
+<h1 class="dcf-txt-h3">Welcome to Innovation Studio Manager</h1>
 
 <h5>Hello, <%= @user.full_name %> (<%= @user.username %>)</h5>
 
-<h4>My Reservations</h4>
+<h4 id="my-reservations">My Reservations</h4>
```
Comment (skoolbus39) [inline views/home.erb:9]: Should probably be an `<h2>` since it immediately follows an `<h1>`

File: views/maker_request_how_to.erb [hunk 3]
```diff
@@ -1,7 +1,5 @@
 
-<div id="pagetitle">
-<h3>Maker Request How To</h3>
-</div>
+<h1 class="dcf-txt-h3">Maker Request How To</h1>
 
 <h4>Maker Request NISM Member Navigation</h4>
```
Comment (skoolbus39) [inline views/maker_request_how_to.erb:4]: Should probably be an `<h2>` since it immediately follows an `<h1>`

File: views/me.erb [hunk 4]
```diff
@@ -1,42 +1,45 @@
-<div id="pagetitle">
-    <h3>Manage My Account</h3>
-</div>
+<h1 class="dcf-txt-h3">Manage My Account</h1>
 
-<p><a href="/home/">View your scheduled events and reservations</a></p>
+<p class="dcf-txt-sm"><a href="/home/">View your scheduled events and reservations</a></p>
 
+<h4>Update Profile</h4>
```
Comment (skoolbus39) [inline views/me.erb:5]: Should probably be an `<h2>` since it immediately follows an `<h1>`

File: views/me.erb [hunk 5]
```diff
@@ -1,42 +1,45 @@
...
 </div>
 </section>
 </form>
 
-<div>
-	<h4>Change Password</h4>
-</div>
-
-<form action="/change-password/" method="POST">
+<h4 class="dcf-mt-6">Change Password</h4>
```
Comment (skoolbus39) [inline views/me.erb:29]: Should this also be an `<h2>`?

File: views/sop_training_document.erb [hunk 6]
```diff
@@ -1,9 +1,7 @@
...
-<h3>SOP
+<h1 class="dcf-txt-h3">SOP
     <% if has_permission?(Permission::MANAGE_RESOURCES) %>
     (<a href="https://innovationstudio.unl.edu/user/login?destination=node/<%= drupal_node_lookup(:sop_training_doc) %>/edit" target="_blank">Edit</a>)
     <% end %>
-</h3>
-</div>
+</h1>
 
 <%= content %>
```
Comment (skoolbus39) [inline views/sop_training_document.erb:7]: No newline at end of file

File: views/tips_and_tricks_training_document.erb [hunk 7]
```diff
@@ -1,9 +1,7 @@
...
-<h3>Tips and Tricks
+<h1 class="dcf-txt-h3">Tips and Tricks
     <% if has_permission?(Permission::MANAGE_RESOURCES) %>
     (<a href="https://innovationstudio.unl.edu/user/login?destination=node/<%= drupal_node_lookup(:tips_training_doc) %>/edit" target="_blank">Edit</a>)
     <% end %>
-</h3>
-</div>
+</h1>
 
 <%= content %>
```
Comment (skoolbus39) [inline views/tips_and_tricks_training_document.erb:7]: No newline at end of file

File: views/tour_signup.erb [hunk 8]
```diff
@@ -1,17 +1,21 @@
-<div id="pagetitle">
-	<h3>Sign Up For <%= event.title %>
-	<span class="dcf-subhead"><%= event.start_time.in_time_zone.strftime('%m/%d/%Y @ %l:%M %P') %></span></h3>
-</div>
+<h1 class="dcf-txt-h3">
+    Sign Up For <%= event.title %>
+	<span class="dcf-subhead"><%= event.start_time.in_time_zone.strftime('%m/%d/%Y @ %l:%M %P') %></span>
+</h3>
```
Comment (skoolbus39) [inline views/tour_signup.erb:4]: Closing tag (`</h3>`) doesn't match the opening tag (`<h1>`)

File: views/workshops_training_document.erb [hunk 9]
```diff
@@ -1,9 +1,7 @@
...
-<h3>Workshops
+<h1 class="dcf-txt-h3">Workshops
     <% if has_permission?(Permission::MANAGE_RESOURCES) %>
     (<a href="https://innovationstudio.unl.edu/user/login?destination=node/<%= drupal_node_lookup(:workshops_training_doc) %>/edit" target="_blank">Edit</a>)
     <% end %>
-</h3>
-</div>
+</h1>
 
 <%= content %>
```
Comment (skoolbus39) [inline views/workshops_training_document.erb:7]: No newline at end of file

File: views/admin/agenda.erb [hunk 10]
```diff
@@ -1,21 +1,15 @@
...
-<div id="pagetitle">
-	<h3>Today's Agenda <span class="dcf-subhead"><%= date.strftime('%B %d, %Y') %></h3>
-</div>
+<h1 class="dcf-txt-h3">Today's Agenda <span class="dcf-subhead"><%= date.strftime('%B %d, %Y') %></h1>
 
-
-<h4>
-Today's Hours
-</h4>
+<h4>Today's Hours</h4>
```
Comment (skoolbus39) [inline views/admin/agenda.erb:12]: Should probably be an `<h2>` since it immediately follows an `<h1>`

File: views/admin/agenda.erb [hunk 11]
```diff
@@ -26,13 +20,11 @@ Today's Hours
     The studio is open all day.
 <% end %>
 
-<h4>
-Today's Reservations
-</h4>
+<h4 id="todays-reservations">Today's Reservations</h4>
```
Comment (skoolbus39) [inline views/admin/agenda.erb:23]: Should this also be an `<h2>`?

File: views/admin/agenda.erb [hunk 12]
```diff
@@ -73,14 +65,12 @@ No reservations today. Hopefully someone will still come in to use the space!<br
 </table>
 <% end %>
 
-<h4 class="dcf-mt-4">
-Today's Events
-</h4>
+<h4 class="dcf-mt-4" id="todays-events">Today's Events</h4>
```
Comment (skoolbus39) [inline views/admin/agenda.erb:68]: Should this also be an `<h2>`?

File: views/admin/emails.erb [hunk 13]
```diff
@@ -1,10 +1,4 @@
...
-	</h3>
-</div>
+<h1 class="dcf-txt-h3">Innovation Studio Email</h1>
 
-<h4>
-Send Email to User(s)
-</h4>
-<a class="dcf-btn dcf-btn-primary" href="/admin/email/send/">Compose</a>
\ No newline at end of file
+<h4>Send Email to User(s)</h4>
```
Comment (skoolbus39) [inline views/admin/emails.erb:3]: Should probably be an `<h2>` since it immediately follows an `<h1>`

**Review by skoolbus39 — CHANGES_REQUESTED** [review]

**Pull request merged**